sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/utils/date-utils.ts

```diff
@@ -0,0 +1,10 @@
+import * as dayjs from "dayjs";
+
+/**
+ * Checks if a string has a valid date timestamp file format.
+ * @param timestamp timestamp to be validated.
+ * @returns true if the format is the expected.
+ */
+export function isValidFileTimestamp(timestamp: string): boolean {
```

Comment:
Just a thought. should this particular util live outside IER scope as I see it could be use for ECE or may be other integrations

Reply:
I had the same thought but ended up adding it here because it is mainly about checking the format `YYYY-MM-DD_HH.mm.ss` that changes for different integrations. I just thought that it was not worth it.
Either way, I am changing since it is used beyond IER12.

Reply:
File moved.